tools/wip.toml

```diff
@@ -0,0 +1,1217 @@
+################################################################################
```

Comment:
Can we move the details to the `Config` class and put a summary here.  The structure -- matching the Feature/Section/Scenario structure of the source is a really, really good idea.

Reply:
Done.